syncer/batchsync.go

```diff
@@ -156,6 +156,8 @@ func syncJobsStatus(storer jobs.Storer, status string, queues []string, job_summ
 				var run_started_time *time.Time
 				var log_stream_name *string
 				var task_arn *string
+				var log_group_name *string
+				log_group_name = desc.Container.LogConfiguration.Options["awslogs-group"]
```

Comment:
I'm getting the panic below when I run `docker-compose up` - maybe because I haven't configured awslogs-group?

```
api_1         | 18:43:54 app         | panic: runtime error: invalid memory address or nil pointer dereference
api_1         | [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x64b5c0]
api_1         |
api_1         | goroutine 24 [running]:
api_1         | 18:43:54 app         | github.com/AdRoll/batchiepatchie/syncer.syncJobsStatus({0xffff89b28098, 0x40000acd68}, {0x94eb22, 0x6}, {0x40000200d0, 0x1, 0x40?}, 0x2?, {0xc68d00, 0x12571a0})
api_1         | 	/go/src/github.com/AdRoll/batchiepatchie/syncer/batchsync.go:161 +0x1000
api_1         | github.com/AdRoll/batchiepatchie/syncer.syncJobs({0xffff89b28098, 0x40000acd68}, {0x40000200d0, 0x1, 0x1})
api_1         | 	/go/src/github.com/AdRoll/batchiepatchie/syncer/batchsync.go:272 +0x1f8
api_1         | github.com/AdRoll/batchiepatchie/syncer.RunSynchronizer({0xc6a0d0?, 0x40000acd68
api_1         | 18:43:54 app         | }, {0x40000200d0, 0x1, 0x1})
api_1         | 	/go/src/github.com/AdRoll/batchiepatchie/syncer/batchsync.go:359 +0x5c
api_1         | github.com/AdRoll/batchiepatchie/syncer.RunPeriodicSynchronizer.func1()
api_1         | 	/go/src/github.com/AdRoll/batchiepatchie/syncer/batchsync.go
api_1         | 18:43:54 app         | :344 +0x168
api_1         | created by github.com/AdRoll/batchiepatchie/syncer.RunPeriodicSynchronizer
api_1         | 	/go/src/github.com/AdRoll/batchiepatchie/syncer/batchsync.go:320 +0xb0
```